vendor/tests/test_stripe.py

```diff
@@ -515,7 +517,70 @@ def test_check_price_does_exist(self):
         self.processor.stripe_delete_object(self.processor.stripe.Price, stripe_price.id)"""
 
     def test_sync_customers(self):
-        pass
+        # Vendor objects not in stripe so create them there
+        user1 = User.objects.create(email='test_email@aol.com', first_name='test name', last_name='last',
+                                    username='test1')
+        customer1 = CustomerProfile.objects.create(site=self.site, user=user1)
+        user2 = User.objects.create(email='test_email2@aol.com', first_name='test name2', last_name='last',
+                                    username='test2')
+        customer2 = CustomerProfile.objects.create(site=self.site, user=user2)
```

Comment:
I don't know if this is setting of the save() signal to sync this new users. But I think it woul be more easy if you add them to the fixtures. I am not sure if when the fixutres loads it triggers a save() signal on the models

Reply:
fixtures do